Source/WebKit/Shared/API/Cocoa/_WKRemoteObjectRegistry.mm

```diff
@@ -297,25 +297,25 @@ - (void)_invokeMethod:(const WebKit::RemoteObjectInvocation&)remoteObjectInvocat
         };
 
         RefPtr<ReplyBlockCallChecker> checker = ReplyBlockCallChecker::create(self, replyID);
-        id replyBlock = __NSMakeSpecialForwardingCaptureBlock([wireBlockSignature UTF8String], [interface, remoteObjectRegistry, replyID, checker](NSInvocation *invocation) {
+        auto replyBlock = adoptNS(__NSMakeSpecialForwardingCaptureBlock([wireBlockSignature UTF8String], [interface, remoteObjectRegistry, replyID, checker](NSInvocation *invocation) {
```

Comment:
I guess this was a leak?

Reply:
No, it was calling release explicitly in this old code.